docs/sources/flow/reference/components/prometheus.exporter.kafka.md

```diff
@@ -27,30 +27,42 @@ prometheus.exporter.kafka "LABEL" {
+| ----------------------------- | --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- | -------- |
+| `kafka_uris`                  | `array(string)` | Address array (host:port) of Kafka server.                                                                                                                                         |         | yes      |
+| `instance`                    | `string`        | The`instance`label for metrics, default is the hostname:port of the first kafka_uris. You must manually provide the instance value if there is more than one string in kafka_uris. |         | no       |
+| `use_sasl`                    | `bool`          | Connect using SASL/PLAIN.                                                                                                                                                          |         | no       |
+| `use_sasl_handshake`          | `bool`          | Only set this to false if using a non-Kafka SASL proxy.                                                                                                                            | `true`  | no       |
+| `sasl_username`               | `string`        | SASL user name.                                                                                                                                                                    |         | no       |
+| `sasl_password`               | `string`        | SASL user password.                                                                                                                                                                |         | no       |
+| `sasl_mechanism`              | `string`        | The SASL SCRAM SHA algorithm sha256 or sha512 as mechanism.                                                                                                                        |         | no       |
+| `sasl_disable_pafx_fast`      | `bool`          | Configure the Kerberos client to not use PA_FX_FAST.                                                                                                                               |         | no       |
+| `use_tls`                     | `bool`          | Connect using TLS.                                                                                                                                                                 |         | no       |
+| `tls_server_name`             | `string`        | Used to verify the hostname on the returned certificates unless tls.insecure-skip-tls-verify is given. The kafka server's name should be given.                                    |         | no       |
```

Comment:
```suggestion
| `tls_server_name`             | `string`        | Used to verify the hostname on the returned certificates unless tls.insecure-skip-tls-verify is given. The Kafka server's name should be given.                                    |         | no       |
```

Should or must? Should implies you don't need to... what happens if you don't provide the Kafka server name?

Reply:
(I did not test it locally because it's a bit annoying to set up)